frontend/src/components/MapView/MapChat/index.js

```diff
@@ -108,66 +105,69 @@ class MapChat extends Component {
   }
 
   render = () => {
-    const rightOffset = this.state.open ? '0' : '-300px'
+    const rightOffset = this.props.chatOpen ? '0' : '-300px'
```

Comment:
I think this method of hiding it is what sometimes causes Tab keypresses to screw up the map view. I've seen this a few times, and i think Rob has too. Better to hide it. Maybe we can take now as our chance to fix that bug